torchdr/utils/geometry.py

```diff
@@ -17,6 +17,7 @@
 
 def pairwise_distances(
     X: torch.Tensor,
```

Comment:
so as to keep only changes here, and rename the PR "towards support for `transform` in affinities where it is possible to transform new data"